master/internal/api_experiment.go

```diff
@@ -595,10 +595,6 @@ func (a *apiServer) PatchExperiment(
 				req.Experiment.Labels)
 		case path == "description":
 			exp.Description = req.Experiment.Description
-		case !strings.HasPrefix(path, "update_mask"):
```

Comment:
reading [this](https://determined-community.slack.com/archives/CV3MVAQJU/p1642614972035700?thread_ts=1641856211.020200&cid=CV3MVAQJU) did you mean to leave this out?
I know it'll limit the changes you can make to the cli and prevent removing parts of the old patch endpoint. This works given/if we have some future plans to do option 1 or 2 or add new endpoints for what patch experiment does.

If you go forward with this could you update the endpoint comment in proto to include what's updatable and maybe that trying to patch other attributes won't fail as an extra warning?

Reply:
If I leave this part alone, sending the rest of the Experiment object in the request will cause an error here.

Is your suggestion a change to PatchExperimentRequest in proto/src/determined/api/v1/experiment.proto , we don't really say anything there about what fields to send

I think the other option would be changing `bindings.v1Experiment.to_json` to have a `to_patch_json` and only include patch-able fields?

Reply:
> If I leave this part alone, sending the rest of the Experiment object in the request will cause an error here.

true. Maybe I could have put it better but my original assumption was that by option0 you're gonna leave out any changes to this but then considering that
> I know it'll limit the changes you can make to the cli and prevent removing parts of the old patch endpoint.

I agreed with going forward with it : )
> This works for me given/if we have some future plans to do option 1 or 2 or add new endpoints for what patch experiment does.

> Is your suggestion a change to PatchExperimentRequest in proto/src/determined/api/v1/experiment.proto , we don't really say anything there about what fields to send

I'm just suggesting adding more details to the comment here. if you go with the current approach you're taking clarifying a 200 doesn't mean their request was necessarily fulfilled.
```
  // Patch an experiment's fields.
  rpc PatchExperiment(PatchExperimentRequest)
```

> I think the other option would be changing bindings.v1Experiment.to_json to have a to_patch_json and only include patch-able fields?

that could be a good option but did you wanna leave discussing and addressing that for another pr? [ref](https://determined-community.slack.com/archives/CV3MVAQJU/p1642631609038600?thread_ts=1641856211.020200&cid=CV3MVAQJU)